.gitignore

```diff
@@ -264,3 +264,6 @@ paket-files/
 __pycache__/
 *.pyc
 /readme_images
+
+# Ignore the VS Directory.build.props file
+Directory.Build.props 
```

Comment:
there is a section starting at line 8 with user specific files including VS stuff, move there?!